docs/tasks/regionalTSDiagrams.rst

```diff
@@ -0,0 +1,220 @@
+.. _task_regionalTSDiagrams:
+
+regionalTSDiagrams
+==================
+
+An analysis task for plotting T-S (potential temperature vs. salinity)
+diagrams of climatologis in ocean regions.
+
+Component and Tags::
+
+  component: ocean
+  tags: climatology, regions, publicObs
+
+Configuration Options
+---------------------
+
+The following configuration options are available for this task::
+
+    [regionalTSDiagrams]
+    ## options related to plotting time series of groups of ocean regions
```

Comment:
update line above (and a couple more similar lines below)

Reply:
Thanks!  Fixed.